PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
```

Comment:
I really wasn't sure what the difference was between this and `Testing done`. Seems like this was for planning future testing, but if the tests didn't make it into a PR, they're probably not going to make it out of the PR.

It could be useful for things that need UAT or something, but since things like that are likely the exceptions, it didn't make sense to include a section for it in _every_ PR.